client/my-sites/domains/domain-management/list/domains-table.jsx

```diff
@@ -77,14 +76,6 @@ class DomainsTable extends PureComponent {
 		return <QuerySitePurchases key={ `query-purchases-${ blogId }` } siteId={ blogId } />;
 	}
 
-	renderQuerySiteDomainsOnce( blogId ) {
-		if ( this.renderedQuerySiteDomains[ blogId ] ) {
-			return null;
-		}
-		this.renderedQuerySiteDomains[ blogId ] = true;
-		return <QuerySiteDomains key={ `query-purchases-${ blogId }` } siteId={ blogId } />;
-	}
```

Comment:
This isn't needed anymore because now all the required domain data is fetched at once